content/en/tracing/guide/add_span_md_and_graph_it.md

```diff
@@ -0,0 +1,273 @@
+---
+title: Add span metadata and graph it using Trace Analytics
+kind: guide
+disable_toc: true
+further_reading:
+- link: "tracing/setup/"
+  tag: "Documentation"
+  text: "Learn how to setup APM tracing with your application"
+- link: "tracing/visualization/services_list/"
+  tag: "Documentation"
+  text: "Discover the list of services reporting to Datadog"
+- link: "tracing/visualization/resource"
+  tag: "Documentation"
+  text: "Dive into your resource performance and traces"
+- link: "tracing/visualization/trace"
+  tag: "Documentation"
+  text: "Understand how to read a Datadog Trace"
+---
+
+{{< img src="tracing/guide/add_span_md_and_graph_it/span_md_6.gif" alt="comparaison 2" responsive="true" style="width:90%;">}}
+
+Datadog APM allows you to customize your traces to include any additional information you might need to maintain observability into your business. In this example a customer ID will be added to traces allowing the customers that have the slowest performance to be identified. Customization of traces is based on tags that seamlessly integrate APM with the rest of Datadog and come in the form of key:value pairs of metadata that is added to spans. You can read more about tags and their use in Datadog in the [Tags Overview][1] page.
+
+## Instrument your code with custom metadata
+
+1. **Follow the example to get your code instrumented**.
+
+    Depending on the language you are you using, you’ll need to set the metadata to add to your spans differently.
```

Comment:
```suggestion
    Depending on the programming language you are you using, you’ll need to set the metadata to add to your spans differently.
```
we have to specify now, since we have docs in more than English